packages/terra-site/themeable-variables.json

```diff
@@ -1,4 +1,4 @@
-{
+x{
```

Comment:
Where did this x come from?

Reply:
not sure, must have been a typo, it's removed